docs/tutorials/fundamentals/quick_start.md

```diff
@@ -48,13 +48,7 @@ You will also see some examples of plugins. The core napari viewer focuses on do
 
 ### Installation
 
-- Download the napari {{ napari_version }} bundled app for simple installation:
-
-    {{ '[Linux installation](https://github.com/napari/napari/releases/download/vNAPARI_VER/napari-NAPARI_VER-Linux-x86_64.zip)'.replace('NAPARI_VER', napari_version) }}<br>
-    {{ '[macOS installation](https://github.com/napari/napari/releases/download/vNAPARI_VER/napari-NAPARI_VER-macOS-x86_64.zip)'.replace('NAPARI_VER', napari_version) }}<br>
-    {{ '[Windows installation](https://github.com/napari/napari/releases/download/vNAPARI_VER/napari-NAPARI_VER-Windows-x86_64.zip)'.replace('NAPARI_VER', napari_version) }}<br>
-
-    *Note: for all other releases, please visit [here](https://github.com/napari/napari/releases) and look for Assets.*
+- Download the napari {{ napari_version }} bundled app for a simple installation. The step-by-step installers are available for macOS, Linux, and Windows in {{ '[the Releases page](https://github.com/napari/napari/releases/vNAPARI_VER)'.replace('NAPARI_VER', napari_version) }}, in the Assets section at the bottom.
```

Comment:
I kinda like the direct links, can we modify them so they work with the new bundle? Perhaps with adding a new subsitution if needed?

Reply:
We have seen some releases with botched version names due to branch/tags shenanigans.  See https://github.com/napari/napari/releases/v0.4.16 for an example. That was the rationale here.

Reply:
Hmm. I think I'd rather cross fingers we don't mess up tags and have direct links. If we mess up we can patch on the website repo?

Reply:
Added links now :)